cr-core/build.gradle

```diff
@@ -138,19 +141,21 @@ checkstyle {
 }
 
 pmd {
+    incrementalAnalysis = false
     ignoreFailures = true
     ruleSetConfig = resources.text.fromArchiveEntry(configurations.codeMetrics, "pmd/pmd.xml")
     ruleSets = []
 }
 
-findbugs {
+spotbugs {
     ignoreFailures = true
     effort = 'max'
     reportLevel = 'medium'
-    toolVersion = '3.0.1'
-    excludeFilterConfig = resources.text.fromArchiveEntry(configurations.codeMetrics, "findbugs/findbugs-exclude.xml")
+    // includeFilter = resources.text.fromArchiveEntry(configurations.codeMetrics, "findbugs/findbugs-exclude.xml")
```

Comment:
I would ask about why this is commented out, but I have a feeling that's just because we've commented it out in other instances of this same code so ... never mind 😅

Reply:
I forgot to mention this in the PR, but as findbugs has been deprecated, and spotbugs (its replacement) has major changes in config. I tried some ways to get it working, but couldnt get it working

Maybe an issue to port the config to new util would be helpful in config repo?